### DISCUSSION TITLE
How about MAUI?

### DISCUSSION THREAD

**Discussion by JaneConan** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#2708] [Q&A] [answered]

**Comment by Keboo** [comment, marked as answer]
At present there are no plans to bring this to Xamarin.Forms or MAUI. Much of this code is tied to WPF so it would be a major undertaking to do. 

Microsoft is much better positioned to implement this with something like this. https://github.com/dotnet/Microsoft.Maui.Graphics.Controls
